<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hw2</class>
 <widget class="QMainWindow" name="hw2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>888</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>hw2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>331</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 239, 239);</string>
    </property>
    <widget class="QLabel" name="lblPicA">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(215, 215, 215);
border-color: rgb(8, 8, 8);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>251</width>
       <height>461</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Mono</string>
      </attribute>
      <widget class="QPushButton" name="btnMono1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Grey Scale1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMono2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Grey Scale2</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>211</width>
         <height>111</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblThresh">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Thresh</string>
        </property>
       </widget>
       <widget class="QSlider" name="hsdThresh">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>125</number>
        </property>
        <property name="sliderPosition">
         <number>125</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="btnThresh">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Threshold</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <widget class="QSlider" name="hsdLevel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="value">
         <number>17</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="lblLevel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLevel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set GreyLevel</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnSubtract">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Subtraction</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <widget class="QSlider" name="hsdResolution">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>25</number>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="lblResolution">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnResolution">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Resolution</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Image Processing</string>
      </attribute>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>181</width>
         <height>111</height>
        </rect>
       </property>
       <widget class="QSlider" name="hsdBrightness">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>160</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBrightness">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Brightness</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblBrightness">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>181</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QPushButton" name="btnContrast">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>101</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Contrast</string>
        </property>
       </widget>
       <widget class="QSlider" name="hsdContrast">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>160</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="lblContrast">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Contrast</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <widget class="QPushButton" name="btnHistoEqualize">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Histogram Equalization</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnOpenA">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Open file A</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblPicA_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>560</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215);
border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblPicA_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215);
border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblPicA_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 215, 215);
border-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>811</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="lyt1">
     <property name="spacing">
      <number>7</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>290</y>
      <width>811</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="lyt2"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>811</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="lyt3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
